test/libyul/Parser.cpp

```diff
@@ -245,16 +245,6 @@ BOOST_AUTO_TEST_CASE(optional_types)
 	BOOST_CHECK(successParse("{ function f(a:u256) -> b {} }"));
 }
 
-BOOST_AUTO_TEST_CASE(invalid_types)
```

Comment:
Why was this removed?

Reply:
Ah looks like they were extracted.

Reply:
I turned it into a YulSyntaxTest, it was easier to do there.